samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -1101,8 +1101,10 @@ int main(void)
 		}
 		printk("Streams started\n");
 
-		/* Start send timer */
-		k_work_schedule(&audio_send_work, K_MSEC(0));
+		if (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT) {
```

Comment:
Good catch on the `IS_ENABLED` - But do consider 
```suggestion
		if (CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT > 0) {
```